translations/en/module_en_lang.php

```diff
@@ -74,6 +75,7 @@
     'wd_credit' => 'Refund',
     'wd_creditor' => 'Creditor',
     'wd_creditor_mandate_id' => 'Mandate ID',
+    'wd_currency_config' => 'Each currency configuration has to be filled out.',
```

Comment:
I would prefer to change this to "Each currency has to be configured." to make it more consistent with the DE version.

Reply:
👍